src/modules/InviteUser/components/InviteUser.vue

```diff
@@ -0,0 +1,379 @@
+              >
+                <img src="../../../assets/x-lg.png" alt="" class="x-lg" />
+              </button>
+            </div>
+          </div>
+          <div class="row w-100 d-flex justify-content-center">
+            <BaseAlert :submission="submission" />
+          </div>
+          <!-- <form @submit.prevent="handleSubmit"> -->
+          <div class="input-frame">
+            <div class="input-with-label">
+              <div class="label">Full Name</div>
+              <input
+                type="text"
+                placeholder="Username"
+                class="input-text"
+                v-model="name.value"
+                @keyup="validateField('name')"
+              />
+              <div
+                v-if="name.error"
+                class="form-text text-danger"
+                v-text="name.error"
+              ></div>
+            </div>
+            <div class="input-with-label">
+              <div class="label">Email</div>
+              <input
+                type="email"
+                placeholder="ex: myworkmail@gmail.com"
+                class="input-text"
+                v-model="email.value"
+                @keyup="validateField('email')"
+              />
+              <div
+                v-if="email.error"
+                class="form-text text-danger"
+                v-text="email.error"
+              ></div>
+            </div>
+            <div class="input-with-label">
+              <div class="label">Role</div>
+              <select
+                class="input-text"
+                aria-label="Default select example"
+                v-model="role.value"
+              >
+                <!-- role list is provided from backend for proper implementation -->
+                <option selected disabled>Select a Role</option>
+                <option data-cy="invite-select-employee" value="1">
+                  Employee
+                </option>
+                <option data-cy="invite-select-vendor" value="2">Vendor</option>
+              </select>
+            </div>
+          </div>
+          <div class="button-frame">
+            <div class="outline-button">
+              <button
+                class="primary button"
+                type="button"
+                data-cy="invite-cancel-btn"
+                data-bs-dismiss="modal"
+              >
+                Cancel
+              </button>
+            </div>
+            <div class="invite-button">
+              <button
+                class="primary-1 button-1"
+                data-cy="invite-send-btn"
+                @click="handleSubmit"
+              >
+                Send Invitation
+              </button>
+            </div>
+          </div>
+          <!-- </form> -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style>
+@import url("https://fonts.googleapis.com/css2?family=Cedarville+Cursive&family=Poppins:wght@200;300&family=Roboto+Slab:wght@300&display=swap");
```

Comment:
Why the font is imported in a component instead of a global CSS file?

Reply:
proper font will be reimported and will be applied globally